CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.13)
+
+option(FIZZY_TESTING "Enable Fizzy internal tests" OFF)
```

Comment:
Enables internal testing. When developing fizzy, this should be set to `ON`, but users can use default options values.